lib/states/base_state.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module States
+  class BaseState
+    EXIT_COMMAND = I18n.t('base_state.exit_command')
+    YES = I18n.t('base_state.apply')
```

Comment:
Why did you put commands in I18? Are we going to use Chinese characters or Cyrillic for this?